discord/ext/commands/context.py

```diff
@@ -484,6 +484,8 @@ async def send(
             if not self.interaction.response.is_done():
                 await self.interaction.response.defer(ephemeral=ephemeral)
 
+            # Webhook.send uses MISSING instead of None
+            kwargs = {k: v if v not in (None, MISSING) else MISSING for k, v in kwargs.items()}
```

Comment:
Why replace `MISSING` with `MISSING`?
```suggestion
            kwargs = {k: MISSING if v is None else v for k, v in kwargs.items()}
```